Gemfile

```diff
@@ -1,10 +1,12 @@
 source 'https://rubygems.org'
-gem 'rake'
+
 gem 'cocoapods', '>= 1.10'
-gem 'xcpretty-travis-formatter'
-gem 'octokit', "~> 4.0"
 gem 'dotenv'
 gem 'fastlane', '~> 2'
+gem 'octokit', '~> 4.0'
+gem 'rake'
```

Comment:
Could we put a version constraint on this while we're changing it?

And we probably don't need `xcpretty-travis-formatter` since we're not using Travis CI anymore, but at the very least we could version-constrain that too?

Reply:
Good point. I only looked at the shape of the diff, not it's actual content 😳

Reply:
> And we probably don't need `xcpretty-travis-formatter` since we're not using Travis CI anymore, but at the very least we could version-constrain that too?

I agree, we shouldn't need it. I just discovered, though, that it's used in the Rake tasks. I think removing it is out of scope for this PR. I opened https://github.com/Automattic/simplenote-macos/issues/1008 to track that.

https://github.com/Automattic/simplenote-macos/blob/f2e2c5bdc7e81e6e1bb6cdbbca82c2bd6198d87a/Rakefile#L289